js/interface.js

```diff
@@ -1,11 +1,5 @@
-var defaultChartHeight = {
-  sm: '400px',
-  md: '500px'
-};
```

Comment:
I assume the new charts won't have different default heights for different breakpoints?

Reply:
@tonytlwu They will, set but the theme, I honoured those values.

Reply:
@tonytlwu 
<img width="358" alt="Screenshot 2019-06-10 at 11 35 31" src="https://user-images.githubusercontent.com/7046481/59190150-e0c4bb00-8b73-11e9-903d-62802aaebf32.png">